dags/data_cleaning_utils/resume_state.py

```
@@ -0,0 +1,94 @@
+                {"_id": ObjectId(self.platform_id)},
+                {"_id": 0, "metadata.resume_index": 1},
+            )
+            if not doc:
+                return 0
+            meta = doc.get("metadata") or {}
+            value = meta.get("resume_index")
+            if value is None:
+                return 0
+            try:
+                return int(value)
+            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)